src/components/tl/mlx5/tl_mlx5.c

```diff
@@ -92,6 +92,10 @@ static ucc_config_field_t ucc_tl_mlx5_lib_config_table[] = {
      ucc_offsetof(ucc_tl_mlx5_lib_config_t, mcast_conf.max_eager),
      UCC_CONFIG_TYPE_MEMUNITS},
 
+    {"MCAST_CUDA_MEM_ENABLE", "0", "Enable GPU CUDA memory support for Mcast. GPUDirect RDMA must be enabled",
+     ucc_offsetof(ucc_tl_mlx5_lib_config_t, mcast_conf.cuda_mem_enabled),
+     UCC_CONFIG_TYPE_INT},
```

Comment:
in the future, this will have a third value, whereby setting that, we enable both HOST and DEVICE versions of MCAST simultaneously. That is why I used an integer, not a boolean.

Reply:
Sure, then let me change the type to BOOL. To enable both host and cuda at same time, I can add a new env var called MCAST_HOST_MEM_ENABLE to toggle the host support.